src/openloco/windows/construction/Common.cpp

```diff
@@ -888,7 +896,7 @@ namespace openloco::ui::windows::construction
                 }
             }
 
-            if (rotation < 12)
+            if (rotation >= 12)
```

Comment:
is this change correct?

Reply:
Yes it fixes a bug reported on discord. The map selection tiles are wrong on diagonal track for 3 of the 4 track rotations, this fixes it.